docker/pypi/wmagent/bin/manage

```diff
@@ -74,7 +74,12 @@ activate_agent(){
     # The very first step of activating the WMAgent upon all needed checks and
     # validations performed during the initialization process.
     # No .init* flags are checked at this stage.
-    cp -vf $WMA_DEPLOY_DIR/etc/WMAgentConfig.py $WMA_CONFIG_DIR/config-template.py
+    if [[ "$TEAMNAME" == Tier0* ]]; then
```

Comment:
Instead of hardcoding a team, how about checking if `$WMA_DEPLOY_DIR/etc/Tier0Config.py` exists?

Reply:
hi @amaltaro  The `Teamname` has been discussed with T0 team and is a valid  criteria for the purpose of distinguishing the production agents  from the Tier0 ones, exactly the same way how we distinguish the `testbed` from `prod` agents.  The `Tier0Config.py` though is not such a reliable criteria mostly because it get installed quite  late in the process, and it comes with the T0 package.